src/Override/OverrideBehaviour.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace ConfigCat\Override;
+
+/**
+ * Class OverrideBehaviour Describes how the overrides should behave.
+ * @package ConfigCat
+ */
+class OverrideBehaviour
+{
+    /**
+     * With this mode, the SDK won't fetch the flags & settings from the ConfigCat CDN, and it will use only the local
+     * overrides to evaluate values.
+     */
+    const LOCAL_ONLY = 10;
+    /**
+     * With this mode, the SDK will fetch the feature flags & settings from the ConfigCat CDN, and it will replace
+     * those that have a matching key in the flag overrides.
+     */
+    const LOCAL_OVER_REMOTE = 20;
+    /**
+     * With this mode, the SDK will fetch the feature flags & settings from the ConfigCat CDN, and it will use the
+     * overrides for only those flags that doesn't exist in the fetched configuration.
+     */
+    const REMOTE_OVER_LOCAL = 30;
```

Comment:
Én ezt javaslom: When evaluating values, the SDK will use all feature flags & settings that are downloaded from the ConfigCat CDN, plus all feature flags & settings that are loaded from local-override sources.
If a feature flag or a setting is defined both in the fetched and the local-override source then the fetched version will take precedence.